straxen/plugins/peaklet_processing.py

```diff
@@ -680,8 +588,17 @@ def setup(self):
                                                      self.config['gain_model'])
 
     def infer_dtype(self):
-        return strax.unpack_dtype(self.deps['peaklets'].dtype_for('peaklets'))
-
+        # wrong order
+        dtype = (strax.unpack_dtype(self.deps['peaklets'].dtype_for('peaklets'))
+                 +[('type_bayse', np.int8, 'Bayes peak classification type')]
```

Comment:
Also can we correct this typo? Causing mailbox errors.

Reply:
good catch, thanks should be fixed now